ios/RNCallKeep/RNCallKeep.m

```diff
@@ -92,10 +97,36 @@ - (void)dealloc
         RNCallKeepPerformPlayDTMFCallAction,
         RNCallKeepDidToggleHoldAction,
         RNCallKeepProviderReset,
-        RNCallKeepCheckReachability
+        RNCallKeepCheckReachability,
+        RNCallKeepDidLoadWithEvents
     ];
 }
 
+- (void)startObserving
```

Comment:
where who to call the `startObserving`?

Reply:
@zxcpoiu React Native calls this method internally for classes which inherit from RCTEventEmitter, see here: https://github.com/facebook/react-native/blob/d0871d0a9a373e1d3ac35da46c85c0d0e793116d/React/Modules/RCTEventEmitter.m#L99 and here: https://reactnative.dev/docs/native-modules-ios#optimizing-for-zero-listeners

Reply:
ah, I just missed this part, thank you！